src/sdk/src/mutation_sdk.rs

```diff
@@ -210,6 +213,23 @@ impl MutationSDK {
         let db_id = DbId::try_from((&sender, nonce))?;
         Ok((db_id, tx_id))
     }
+
+    pub async fn fetch_block_by_height(&self, height: u64) -> Result<BlockResponse> {
```

Comment:
move this method to store_sdk

1. read operation goes into store_sdk
2. write operation goes into mutation_sdk

Reply:
resolved in the 2nd revision